src/Enclave.Sdk/Clients/AuthorityClient.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Net.Http.Json;
+using Enclave.Sdk.Api.Clients.Interfaces;
+using Enclave.Sdk.Api.Data.Authority;
+
+namespace Enclave.Sdk.Api.Clients;
+
+/// <inheritdoc cref="IAuthorityClient" />
+internal class AuthorityClient : ClientBase, IAuthorityClient
+{
+    private readonly string _orgRoute;
+
+    /// <summary>
+    /// Constructor  which will be called by <see cref="OrganisationClient"/> when it's created.
+    /// </summary>
+    /// <param name="httpClient">an instance of httpClient with a baseURL referencing the API.</param>
+    /// <param name="orgRoute">The organisation API route.</param>
+    public AuthorityClient(HttpClient httpClient, string orgRoute)
```

Comment:
I'm conscious of the fact that you do not need a known org in order to use the authority. Also, where do I get an instance of this client?

Reply:
ah i haven't added the instance to the org client 😬

Reply:
You're right you don't need an org i suppose this could be something that the enclave client generates?

Reply:
That seems possible; worth noting also that you don't need an authorisation token; not sure if you have a way of handling that.